video_mapper.go

```diff
@@ -326,7 +331,7 @@ func unsafeJSONMarshal(v interface{}) ([]byte, error) {
 	return b, nil
 }
 
-func isPublishEvent(video map[string]interface{}) (publishEvent bool, err error) {
+func isPublishEvent(video map[string]interface{}) (_ bool, err error) {
```

Comment:
I don't think it has an advantage using _ for named return type here. 
Maybe it's even simpler if we don't use named return types here at all, cause we don't benefit neither from the default init it offers, neither from the return clause.